plaso/engine/processing_status.py

```diff
@@ -508,6 +518,19 @@ def UpdateWorkerStatus(
         number_of_consumed_reports, number_of_produced_reports)
 
 
+class EventsStatus(object):
+  """The status of the events.
+
+  Attributes:
+    total_number_of_events (int): total number of events.
```

Comment:
Is this intended to be the total number of events in a store, or just a number of events in general? If the latter, drop the "total_", if something else, please update the docstring to clarify what this is a total of.

Reply:
Ping for @joachimmetz - I think you missed this comment

Reply:
Extra comment to add to code review

Reply:
Total in storage file.